src/components/auction/OrderbookWidget/index.tsx

```diff
@@ -140,8 +140,8 @@ const processData = (
         // Data for representation
         priceNumber: price,
         totalVolumeNumber: totalVolume,
-        priceFormatted: price.toFixed(2),
-        totalVolumeFormatted: totalVolume.toFixed(2),
+        priceFormatted: price.toFixed(5),
+        totalVolumeFormatted: totalVolume.toFixed(5),
```

Comment:
Not sure about this, but it possible that a constant like `MAX_DECIMALS_PRICE_FORMAT` or similar could clear the intent and make any refactor/change easier to do.